### ISSUE TITLE
How to prepare a react-vis dev repo?

### ISSUE THREAD

**Issue by talc23** [uber/react-vis#1141]
Hi,
I'm using react-vis for my side project, and I want to contribute to this project.
However, it's not clear to me how to prepare a local repo where I can implement the changes I want to commit.

The readme holds this:
> Development
To develop on this component, install the dependencies and then build and watch the static files:
npm install && npm run start

What should be done prior to running npm install?
should I create a react project and install react-vis thru npm?
should I clone the repo and run npm install?
what are the "dependencies"?

**Comment by mcnuttandrew** [comment]
Hey @talc23,
A good way to  contribute to this project is by forking it (see button in upper right hand corner), cloning your newly forked repo, navigating into that repo in via the command line and then running npm install && npm install run start. This will install dependencies and dev-dependencies listed in package.json (like most other modern js projects react-vis has a quite a few). Make whatever changes you want local and push them up to your fork using git push. Then navigate to the original react-vis (ie this page) and look for the "open new pr" button and work as appropriate. For more details, check out other open source contribution guides. 
Best
